docs/glossary.md

```diff
@@ -0,0 +1,25 @@
+# Glossary
+
+
+## Agent
+## Bundle
+## Chart
+## Cluster
+## Continuous Delivery
+## Custom Resource
+## Custom Resource Definition
+## Deployment
+## Downstream Cluster
+## fleet.yaml
+## GitOps
+## GitRepo
+## Label
+## Multi-Cluster
+## Namespace
+## Registration
+## Repository
+## Resources
+## Target
+## Upstream Cluster
+## Workload
```

Comment:
```suggestion

## Workload

A workload represents what users want to deploy through Fleet. It may be a set of Helm charts, Kubernetes manifests, kustomize, etc, stored or referenced in a git repository.

When a user creates a `GitRepo` resource pointing to that git repository, and subsequently when relevant changes are found in that repository, Fleet deploys workloads.
```